src/main/resources/application.properties

```
@@ -3,12 +3,14 @@
 # ===============================
 server.port=8080
 
+spring.config.import=optional:file:.env[.properties]
+
 # ===============================
 # Database (PostgreSQL)
 # ===============================
-spring.datasource.url=jdbc:postgresql://localhost:5432/{database_name}
-spring.datasource.username={your_username}
-spring.datasource.password={your_password}
+spring.datasource.url={DB_URL}
+spring.datasource.username={DB_USERNAME}
+spring.datasource.password={DB_PASSWORD}
```

달려 표시가 있어야 환경 변수로 인식 됩니다!

```suggestion
spring.datasource.url=${DB_URL}
spring.datasource.username=${DB_USERNAME}
spring.datasource.password=${DB_PASSWORD}
```